CONTRIBUTING.md

```diff
@@ -34,83 +30,96 @@ If no existing Galaxy issue seems appropriate, a new issue can be opened using
 
 ## How to Contribute
 
-* All changes to the [core galaxy
-  project](https://github.com/galaxyproject/galaxy) should be made through pull
-  requests to this repository (with just two exceptions outlined below).
+All changes to the [core galaxy
+repository](https://github.com/galaxyproject/galaxy) should be made through pull
+requests (with just two exceptions outlined below).
 
-* If you are new to Git, the [Try Git](http://try.github.com/) tutorial is a
-  good places to start.  More learning resources are listed at
-  https://help.github.com/articles/good-resources-for-learning-git-and-github/
+If you are new to Git, the Software Carpentry's [Version Control with
+Git](https://swcarpentry.github.io/git-novice/) tutorial is a good places to
+start.  More learning resources are listed at
+https://help.github.com/en/github/getting-started-with-github/git-and-github-learning-resources
 
-* Make sure you have a free [GitHub](https://github.com/) account.
+1. Make sure you have a free [GitHub](https://github.com/) account.
 
-* Fork the [galaxy repository](https://github.com/galaxyproject/galaxy) on
-  GitHub to make your changes.  To keep your copy up to date with respect to
-  the main repository, you need to frequently [sync your
-  fork](https://help.github.com/articles/syncing-a-fork/):
-  ```
-    $ git remote add upstream https://github.com/galaxyproject/galaxy
-    $ git fetch upstream
-    $ git checkout dev
-    $ git merge upstream/dev
-  ```
+2. Fork the [galaxy repository](https://github.com/galaxyproject/galaxy) on
+   GitHub to make your changes.  To keep your copy up to date with respect to
+   the main repository, you need to frequently [sync your
+   fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork):
 
-* Choose the correct branch to develop your changes against.
+   ```
+   $ git remote add upstream https://github.com/galaxyproject/galaxy
+   $ git fetch upstream
+   $ git checkout dev
+   $ git merge upstream/dev
+   ```
 
-  * Additions of new features to the code base should be pushed to the `dev`
-    branch (`git checkout dev`).
+3. Choose the correct branch to develop your changes against.
 
-  * Most bug fixes to previously release components (things in galaxy-dist)
-    should be made against the recent `release_XX.XX` branch (`git checkout
-    release_XX.XX`).
+   * Additions of new features to the code base should be pushed to the `dev`
+     branch (`git checkout dev`).
 
-  * Serious security problems should not be fixed via pull request - please see
-    [the Galaxy security policies](SECURITY_POLICY.md) for information about
-    responsibly disclosing security issues.
+   * Most bug fixes to previous (or freezed) releases should be made against the
```

Comment:
```suggestion
   * Most bug fixes to previous (or frozen) releases should be made against the
```

Reply:
Ouch, that's what is also used in doc/source/project/organization.rst unfortunately, originally introduced by me :blush: 

Should we keep it as is for consistency until we update the procedure document?

Reply:
Just being practical I'd probably fix it here now and roll the procedures fix into the next amendment there.  But I don't really have a strong opinion either way, up to you.